com.unity.netcode.gameobjects/Tests/Runtime/Transport/SIPTransport.cs

```diff
@@ -13,9 +13,9 @@ namespace Unity.Netcode.RuntimeTests
     /// it's designed for the netcode in a way where no networking stack has to be available
     /// it's designed for testing purposes and it's not designed with speed in mind
     /// </summary>
-#if UNITY_EDITOR
-    [DontShowInTransportDropdown]
-#endif
+    //#if UNITY_EDITOR
```

Comment:
Was leaving this commented out an intentional change? SIPTransport doesn't feel like something that makes sense to be exposed in the editor, even in testproject.

Reply:
this was all changed...this PR will get deleted sometime this week.  The solution was to create a derived class from NetworkTransport that is what SIPTransport derives from.  In the editor it checks for anything that is a subclass of the testing transport class or if the class is derived from the testing transport.